code/controllers/subsystems/managed_instance.dm

```diff
@@ -0,0 +1,47 @@
+SUBSYSTEM_DEF(managed_instances)
+	name = "Managed Instances"
+	flags = SS_NO_FIRE | SS_NO_INIT
+	var/list/_managed_instance_cache = list()
+
+/datum/controller/subsystem/managed_instances/proc/get(var/instance_type, var/cache_id, var/cache_category = "default", var/list/instance_args)
+	if(!cache_id)
+		cache_id = instance_type
+	if(isnull(cache_id)) // No type and no id supplied, no point going further.
+		CRASH("Attempted to retrieve a managed instance with no ID.")
```

Comment:
```suggestion
		CRASH("Attempted to retrieve a managed instance with no type or ID.")
```
Feel like this should be noted, since from just the crash message alone it's unclear that "no ID" means "no type" as well.